### DISCUSSION TITLE
Web UI

### DISCUSSION THREAD

**Discussion by kenwiesner** [raspicamplayer/camplayer#29] [Ideas]
Not sure what would be involved but having a web UI to configure this would be very helpful!

**Comment by adyannu** [comment]
That would be really great!

**Comment by conquesttechnology** [comment]
> Not sure what would be involved but having a web UI to configure this would be very helpful!

I built one using NGINX and PHP so that non-technical users can update the RTSP streams. Examples of V1 and V2 are attached. We monitor the config file and restart the camplayer service when it changes to update without them having to reboot. The current config is displayed so they can copy and past the streams, they want to keep. I only have the 4-channel config page complete since that's what we have needed. 

![GUI V1](https://user-images.githubusercontent.com/85844076/126271598-f64ba1b4-61fe-4165-a9db-7565f66500f4.jpg)
![GUI V2](https://user-images.githubusercontent.com/85844076/126271605-2331bf2a-f6a8-4bb3-a90e-9fca2a5a56a3.jpg)

**Reply by IsaacBreuer** [reply]
whould you mind sharing your come for this. so we can adopt it further?

**Comment by HeliSteve** [comment]
Hello, was this code ever released?

**Comment by dorinclisu** [comment]
https://github.com/dorinclisu/camplayer_streamlit
